todo/mail_worker.py

```diff
@@ -0,0 +1,196 @@
+import email
+import email.parser
+import imaplib
+import logging
+import os
+import sys
+import time
+import yaml
+
+import django
+django.setup()
+
+from argparse import ArgumentParser
+from django.db import transaction
+from django.db.models import Count
+from email.charset import Charset as EMailCharset
+from email.policy import default
+from html2text import html2text
+from todo.models import Comment, EMail, Task, TaskList, Group
+
+
+logger = logging.getLogger(__name__)
+
+
+def imap_check(command_tuple):
+    status, ids = command_tuple
+    assert status == 'OK', ids
+    
+
+def imap_connect(config):
+    conn = imaplib.IMAP4_SSL(host=config.host, port=config.port)
```

Comment:
aren't django settings for sending emails ?
what would be the format of this ? as we can't start all workers that would be configured in the django config, is it actually a good idea ?

Reply:
alright, maybe a `Dict[worker_name, worker_settings] ` is acceptable

Reply:
> aren't django settings for sending emails ?

The native ones are - this would be a new setting you'd introduce into django-todo. But the code would still pull the actual credentials from django settings file or env vars. So:

`
TODO_EMAIL_SERVER = [fqdn]
TODO_EMAIL_BOX= "a@b.com"
TODO_EMAIL_PASS = "XYZ"
...
`

> as we can't start all workers that would be configured in the django config

The worker would not start automatically when django starts - it has to be started manually or during deploy. But it's still nicer for consistency not to have non-Django scripts sitting around. This should be a good citizen of the framework.